telegram/ext/filters.py

```diff
@@ -603,6 +603,66 @@ def filter(self, message: Message) -> bool:
         xml = mime_type('application/xml')
         zip = mime_type('application/zip')
 
+        class file_extension(MessageFilter):
+            """This filter filters documents by their file ending/extension.
+
+            Note:
+                * This Filter only filters by the file ending/extension of the document,
+                    it doesn't check the validity of document.
+                * The user can manipulate the file extension of a document and
+                    send media with wrong types that don't fit to this handler.
+                * Case insensitive by default,
+                    you may change it with the flag ``case_sensitive=True``.
```

Comment:
```suggestion
                    you may change this with the flag ``case_sensitive=True``.
```

this because it is a behaviour